### DISCUSSION TITLE
aggrid style is strange when using left_drawer

### DISCUSSION THREAD

**Discussion by tz301** [zauberzeug/nicegui#2850] [Q&A] [answered]
### Question

I have use aggrid combined with left drawer. When click button to show/hide the left drawer, aggrid style is strange, not fully fill the width.

```py
from nicegui import ui

rows = [
    {'name': 'Alice', 'age': 18},
    {'name': 'Bob', 'age': 21},
]

left_drawer = ui.left_drawer(value=True).classes('bg-blue-100')
ui.button('click', on_click=left_drawer.toggle)

grid = ui.aggrid({
    'columnDefs': [
        {'headerName': 'Name', 'field': 'name'},
        {'headerName': 'Age', 'field': 'age'},
    ],
    'rowData': rows,
})

ui.run()
```

![image](https://github.com/zauberzeug/nicegui/assets/35760938/89a9d7a4-dfd1-4105-bdd2-d1dbb8a67a55)
![image](https://github.com/zauberzeug/nicegui/assets/35760938/a26d94cd-2e6e-4348-a4ff-d62653502a73)

**Comment by falkoschindler** [comment, marked as answer]
Thanks for bringing this up, @tz301!
It seems to be a more general bug with `ui.aggrid`'s resizing mechanism. I created issue #2852.
